docs/components/page_shell.js

```diff
@@ -17,22 +37,177 @@ if (typeof window !== 'undefined' && window.initializeMapboxAnalytics) {
     });
 }
 
+const slugger = new GithubSlugger();
+
 class PageShell extends React.Component {
 
     render() {
+        const { frontMatter, location, children } = this.props;
+        let activeTab = location.pathname.split('/')[2];
+        if (activeTab === 'example') activeTab = 'examples';
+        let topNavContent = (
+            <TopNavTabs 
+                activeTab={activeTab}
+            />
+        );
+        let productName = 'Mapbox GL JS';
+
+        let sidebarContent = <div />;
+        let sidebarStackedOnNarrowScreens = false;
+        if (location.pathname.indexOf('overview') > -1) {
+            sidebarStackedOnNarrowScreens = true;
+            const sections = overviewNavigation.map(section => {
+                return {
+                    title: section.title,
+                    path: `/mapbox-gl-js/overview/${section.path}`
+                }
+            });
+            const subtitles = overviewNavigation.filter(section => {
+                return section.title === frontMatter.title;
+            }).map(section => {
+                return section.subnav.map(subNavItem => {
+                    return {
+                        title: subNavItem.title,
+                        path: `#${subNavItem.path}`
```

Comment:
this is adding double `#` for me (ex. `http://localhost:8080/mapbox-gl-js/overview/##quickstart`)